pkg/gce-pd-csi-driver/node.go

```diff
@@ -38,7 +38,7 @@ type GCENodeServer struct {
 	DeviceUtils     mountmanager.DeviceUtils
 	MetadataService metadataservice.MetadataService
 	// TODO: Only lock mutually exclusive calls and make locking more fine grained
-	mux sync.Mutex
+	volumes sync.Map
```

Comment:
Lets just add a comment here explaining what this sync achieves. Something like "locks so that only one node operation can be done on a single volume (defined by a volume ID) at a time, any concurrent calls will return an AlreadyExists error".